source/slang/slang-parameter-binding.cpp

```diff
@@ -1113,6 +1128,9 @@ static void addExplicitParameterBindings_GLSL(
         semanticInfo.kind = kind;
 
         addExplicitParameterBinding(context, parameterInfo, varDecl.getDecl(), semanticInfo, count);
+        if (foundSubpass)
```

Comment:
Do we actually consume this info?

Reply:
we do use foundSubpass, it is how the overlapping system now functions